### PR TITLE
Updated resources.html

### PR DESCRIPTION
**Pull request by packtpartner**
Added new video to resources

### CODE REVIEW

**Comment by packtpartner** [comment]
Thanks for the merge @lirantal 
When will the video reflect on https://meanjs.org/docs.html#resources ?

File: _includes/docs/0.4.x/resources.html [hunk 1]
```diff
@@ -5,8 +5,14 @@
 <section>
   <p>
    <a href="https://www.packtpub.com/web-development/mean-web-development-second-edition">MEAN Web Development - Second Edition</a>
+    <p>
```
Comment (codydaig) [inline _includes/docs/0.4.x/resources.html:8]: remove

File: _includes/docs/0.4.x/resources.html [hunk 2]
```diff
@@ -5,8 +5,14 @@
...
    <a href="https://www.packtpub.com/application-development/web-application-development-mean">Web Application Development with MEAN</a>
+    <p>
```
Comment (codydaig) [inline _includes/docs/0.4.x/resources.html:10]: remove

File: _includes/docs/0.4.x/resources.html [hunk 3]
```diff
@@ -5,8 +5,14 @@
...
    <a href="https://www.packtpub.com/web-development/mean-blueprints">MEAN Blueprints</a>
+    <p>
```
Comment (codydaig) [inline _includes/docs/0.4.x/resources.html:12]: remove

File: _includes/docs/0.4.x/resources.html [hunk 4]
```diff
@@ -5,8 +5,14 @@
...
    <a href="https://www.packtpub.com/web-development/mastering-mean-web-development-expert-full-stack-javascript-video">Mastering MEAN Web Development: Expert Full Stack JavaScript (Video)</a>
+    <p>
```
Comment (codydaig) [inline _includes/docs/0.4.x/resources.html:14]: remove

File: _includes/docs/0.4.x/resources.html [hunk 5]
```diff
@@ -5,8 +5,14 @@
...
+   <a href="https://www.packtpub.com/application-development/creating-applications-mean-stack-video">Building Applications with Mean Stack [Video]</a>
+    <p>
```
Comment (codydaig) [inline _includes/docs/0.4.x/resources.html:16]: remove

**Review by codydaig — COMMENTED** [review]
Remove the following lines that got added and it should then be proper html and should be good to compile and publish.

**Comment by codydaig** [comment]
There's probably an issue with the html. This isn't proper html. This PR added a bunch of <p> tags but no closing </p> tags. I created a review that notes which lines were excessive.

**Comment by lirantal** [comment]
@packtpartner can you please submit a PR fixing what @codydaig mentioned?

**Comment by packtpartner** [comment]
Thanks for the assistance @codydaig . I have made the changes and submitted a new PR

**Pull request merged**